src/main/g8/README.md

```diff
@@ -261,13 +271,4 @@ object Main extends LazyLogging {
 
 ## Who do I talk to? ##
 
-- Email: chap.s.b@gmail.com
-- Github: https://github.com/sbchapin/
-- Bitbucket: https://bitbucket.org/sbchapin/
-- Discord: sbchapin#7435
-
-## What's the future of this repo? ##
-
-- Keep this baby up to date
-- More robust examples of the existing libraries
-- CI & CD Integrations
```

Comment:
I moved this info up into the README for this project.
https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R67

Reply:
Honestly, I think we can just drop it.  I'll move in the issue of projects / issue tracker if I need a more complex TODO list.